scripts/validate-icons

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+source "$self_dir"/utils.sh
+
+ICONS_PATH='frontend/icons/src/main/resources'
+icon_files=$(git ls-files "$ICONS_PATH/*.svg")
+declared_icons=$(git grep -Pho '(?<=Icon) .* (?=\= loadIcon)' -- 'frontend/icons/src/main/java/icons/MacheteIcons.java')
+
+for referenced_file in $(git grep -Pho '(?<=loadIcon\("/).*(?="\))' -- 'frontend/icons/src/main/java/icons/MacheteIcons.java'); do
+  referenced_file_path="$ICONS_PATH/$referenced_file"
+  referenced_file_path_dark=${referenced_file_path::-4}_dark.svg
+  if [[ ! $icon_files =~ $referenced_file_path ]]; then
+    die "Icon '$referenced_file' referenced from MacheteIcons.java is missing from resources!"
+  fi
+  if [[ ! $icon_files =~ $referenced_file_path_dark ]]; then
+    die "Dark icon '$referenced_file' referenced from MacheteIcons.java is missing from resources!"
```

Comment:
Actually... all these changes to svgs are resizes? 🤔

Reply:
changes to svgs are mostly reformatiings (and some cleanup) - no resizes

Reply:
Oh, xml reformattings 🤦🏻 I keep forgetting that svg is xml